tests/proxy_unit_tests/test_auth_checks.py

```diff
@@ -319,13 +420,11 @@ async def test_virtual_key_max_budget_check(
 
     user_obj = LiteLLM_UserTable(
         user_id="test-user",
-        user_email="test@email.com",
+        user_email="test@example.com",
```

Comment:
FYI this is just an alignment, example.com is used elsewhere throughout this file.